tools/make/lint.mk

```diff
@@ -9,28 +9,22 @@ GITHUB_ACTION ?=
 .PHONY: lint
 lint: ## Run all linter of code sources, including golint, yamllint, whitenoise lint and codespell.
 
-PHONY: lint-deps
-lint-deps: ## Everything necessary to lint (useful to separate out in the logs)
```

Comment:
IMO, when looking at CI logs it's useful to have the linter output not interspersed with ~100 `go: downloading gopkg.in/ini.v1 v1.67.0` lines as things get installed. And so that's why I created a `lint-deps` target.  If you think that value isn't worth the hassle of maintaining the `lint-deps` dependency list, then OK.

Reply:
ah cool, makes sense, I'll close this one